requirements.txt

```diff
@@ -16,3 +16,6 @@ nbmake >= 1.5
 traitlets >= 5.10
 tabulate
 bottleneck >= 1.3.4
+scikit-learn
+jsonschema
+girder_client
```

Comment:
since we use pyproject.toml we do not need requirements anymore we can include the dependencies there